pkg/plugins/runtime/gateway/testdata/http/18-gateway-route.yaml

```diff
@@ -1,13 +1,13 @@
 Clusters:
   Resources:
-    external-httpbin-2528f53d03636b9d:
+    external-httpbin-4a2cc28cd5e983f1:
       circuitBreakers:
         thresholds:
         - maxConnections: 1024
           maxPendingRequests: 1024
           maxRequests: 1024
           maxRetries: 3
-      connectTimeout: 10s
+      connectTimeout: 5s
```

Comment:
Nice. So this comes from the `default-all-timeout` `Timeout` but the `no-timeout` test has `10s` because `default-all-timeout` is deleted.

Should these be different "default" values?

Reply:
It sort of doesn't make sense to have different values, but at the same time I find it convenient that we can see where timeout comes from :)

Reply:
Yeah that's true indeed.